test/test_fakescreen.py

```diff
@@ -171,7 +171,7 @@ def test_basic(manager):
     assert manager.c.screen.info() == {
         'y': 480, 'x': 0, 'index': 2, 'width': 500, 'height': 400}
     manager.c.to_screen(3)
-    manager.test_xclock()
+    manager.test_window('xclock')
```

Comment:
Let's not call it xclock if we don't use xclock anymore. Also it is mentioned in the docs that xclocks is required for the tests, so we should remove that too.

Reply:
Cool - fixed these.